definitions/sonarqube-common.d.ts

```diff
@@ -18,7 +18,7 @@ declare module 'sonarqube-common/sonarqube-common' {
 
     // Applies required parameters for connecting a Java-based plugin (Maven, Gradle) to SonarQube.
     // sqDbUrl, sqDbUsername and sqDbPassword are required if the SonarQube version is less than 5.2.
-    export function applySonarQubeConnectionParams(toolRunner:ToolRunner, sqHostUrl, sqHostUsername, sqHostPassword, sqDbUrl?, sqDbUsername?, sqDbPassword?):ToolRunner;
+    export function applySonarQubeConnectionParams(toolRunner:ToolRunner): ToolRunner;
```

Comment:
We may as well do the getInput calls inside sonarqube-common rather than inside each individual __sonar.ts files.